sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -148,7 +169,29 @@ const ListingForm = ({ listing, editMode }: ListingFormProps) => {
       const typedData: FormListing = {
         ...data,
         yearBuilt: data.yearBuilt ? Number(data.yearBuilt) : null,
+        postmarkedApplicationsReceivedByDate:
+          data.postMarkDate && data.arePostmarksConsidered
+            ? new Date(
+                `${data.postMarkDate.year}-${data.postMarkDate.month}-${data.postMarkDate.day}`
+              )
+            : null,
+        applicationDropOffAddressType:
+          ListingApplicationDropOffAddressType[data.whereApplicationsDroppedOff] ?? null,
+        applicationPickUpAddressType:
+          ListingApplicationPickUpAddressType[data.whereApplicationsPickedUp] ?? null,
+        applicationDropOffAddress:
+          data.canApplicationsBeDroppedOff && data.whereApplicationsPickedUp === "anotherAddress"
+            ? data.applicationDropOffAddress
+            : null,
+        applicationPickUpAddress:
+          data.canPaperApplicationsBePickedUp && data.whereApplicationsPickedUp === "anotherAddress"
+            ? data.applicationPickUpAddress
+            : null,
```

Comment:
What do you think about creating const/enum for `anotherAddress` string? It could prevent string repetition and potential problems :)